timescaledb/index.md

```diff
@@ -16,6 +16,10 @@ to understand what TimescaleDB is, how it works, how to get started, and how
 to successfully use and implement all of the powerful features for your next
 project.
 
+<Tabs>
```

Comment:
Did this creep in here from a different branch?

Reply:
🤦🏼‍♀️ Yep